datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -69,43 +72,6 @@ use parquet::file::{
 };
 use parquet::schema::types::ColumnDescriptor;
 
-#[derive(Debug, Clone, Default)]
-/// Specify options for the parquet scan
-pub struct ParquetScanOptions {
```

Comment:
The key change to this PR is removing this structure and instead reading the values from a `ConfigOptions` that is threaded down.

You can see in this PR there is already a structure for configuring parquet reading (`ParquetReadOptions`) so I actually think this will make the code less confusing to work with going forward.